tests/unit/transforms/test_html.py

```diff
@@ -50,6 +50,30 @@ def test_xpath_selector_all():
     ])
 
 
+def test_xpath_selector_boolean():
+    ok, content = run_transform('xpath', 'boolean(//body/h2/a)', HTML)
+    assert ok is True
+    assert content.strip() == 'True'
```

Comment:
I think it makes sense to move this `.strip()` to `xpath_selector`

Reply:
It actually already is (see line 112 of ``kibitzr/transformer/html.py``) I will just remove it from the tests.